samples/star-wars-api/HttpHandlers.fs

```diff
@@ -37,23 +43,58 @@ module HttpHandlers =
 
     let private graphQL (next : HttpFunc) (ctx : HttpContext) =
         task {
+            let logger = ctx.RequestServices.CreateLogger moduleType
```

Comment:
We can inject logger from DI, like here
https://github.com/giraffe-fsharp/Giraffe/issues/10

Reply:
it already uses `ILoggerFactory`, so it's fine